modules/sovrnBidAdapter.js

```diff
@@ -43,15 +57,30 @@ export const spec = {
           bidfloor: utils.getBidIdParameter('bidfloor', bid.params)
         });
       });
+
+      const page = bidderRequest.refererInfo.referer
+      // clever trick to get the domain
+      const el = document.createElement('a');
+      el.href = page;
+      const domain = el.hostname;
```

Comment:
Instead of doing this (which isn't really ideal), can you use the `parse` function from the `url.js` file to read the `page` variable; it'll provide the `hostname` in the returned object.